### ISSUE TITLE
[Bug]: inaccessible suggestion chip

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ouds-flutter#331]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

any

### OS Version

any

### App version

0.6.0 (361)

### Describe the issue

* open the chip component
* select suggestion chip option
* the page opened is the filter chip

### Expected Behavior

* the page opene should be the suggestion chip

**Comment by AhmedAmineZr** [comment]
@B3nz01d The title is not correct in top of page

**Comment by nouha06** [comment]
@B3nz01d I have checked it, only the title is incorrect, the page itself is correct.

**Comment by AhmedAmineZr** [comment]
📣 New Firebase App Distribution BETA upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (beta) (377)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 331
cc @B3nz01d (product owner)
cc @Tayebsed93 @nouha06 @AhmedAmineZr (dev team)
cc @jerome-regnier @MaximeTonnerre (design team)

**Comment by AhmedAmineZr** [comment]
📣 New TestFlight BETA upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (beta) (377)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 331
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier @MaximeTonnerre (design team)

**Comment by B3nz01d** [comment]
Closed during OUDS sprint # 20
